src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -259,4 +264,24 @@ public void onHide() {
     // Hide all games to free up memory
     filteredItems.setPredicate(game -> false);
   }
+
+  public void onMapsManagementButtonClicked() {
+
+    // TODO: Move button to another space in future
+    ManageMapsController controller = uiService.loadFxml("theme/maps/manage_maps.fxml");
+    FxStage fxStage = FxStage.create(controller.getRoot())
```

Comment:
I would make this a stack pane in the maps vault rather than a new window i think